packages/wrangler/src/dev/remote.tsx

```diff
@@ -122,8 +119,13 @@ export function useWorker(props: {
         logger.log("⎔ Detected changes, restarted server.");
       }
 
+      // Ensure we have an account id, even if it means logging in here.
+      accountIdRef.current = await requireAuth({
```

Comment:
If I understand this correctly, this will be called on every script change, and make the api cal lto fetch account id every time. I think the fix here is to move this outside into it's own effect that runs only once

Reply:
`requireAuth()` does get called every time there is a script change, but this short-circuits because we are passing the `accountIdRef.current` in as the account id.

So there are no extra API calls.

Reply:
Oh you're right!